ansible/library/pulp_facts.py

```diff
@@ -50,27 +50,26 @@ def selinux_enforcing():
 
 
 def available_repositories(module):
-    # determine which of the platform and plugin are available,
-    # failing if platform is not available
-    available_repositories = []
+    """Find the source code for the Pulp platform and its plugins.
 
+    Raise an exception if the source code for the platform is absent.
+    """
+    available_repositories_ = []
```

Comment:
It would be better to name this ```available_repos``` or ```_available_repositories```.  The trailing underscore is easy to overlook.

Reply:
Sure. My intent is to ensure that this variable name doesn't shadow `available_repos()`.